### PR TITLE
buttons: add default button action

### PR DESCRIPTION
**Pull request by helenphan0**

### CODE REVIEW

File: src/editors/button/ButtonEditor.js [hunk 1]
```diff
@@ -19,13 +19,16 @@ export default class ButtonEditor extends React.Component {
 
     const marginTop = persistedState.get('marginTop');
     const marginBottom = persistedState.get('marginBottom');
+    const buttonActionType = persistedState.get('buttonActionType');
 
     const isMarginTopSet = marginTop || marginTop === 0;
     const isMarginBottomSet = marginBottom || marginBottom === 0;
+    const defaultButtonAction = buttonActionType !== 0 && !buttonActionType && BUTTON_ACTION_TYPES.NEXT_PAGE;
```
Comment (GuruM) [inline src/editors/button/ButtonEditor.js]: why does `buttonActionType` ever `= 0`?

Comment (helenphan0) [reply src/editors/button/ButtonEditor.js]: It's when custom page = 1, or data-step=0, which takes you back to the first step in the step group.

Comment (GuruM) [reply src/editors/button/ButtonEditor.js]: wouldn't it be `BUTTON_ACTION_TYPES.CUSTOM_PAGE` for that?

File: src/editors/button/ButtonEditor.js [hunk 2]
```diff
@@ -19,13 +19,16 @@ export default class ButtonEditor extends React.Component {
 
     const marginTop = persistedState.get('marginTop');
     const marginBottom = persistedState.get('marginBottom');
+    const buttonActionType = persistedState.get('buttonActionType');
 
     const isMarginTopSet = marginTop || marginTop === 0;
     const isMarginBottomSet = marginBottom || marginBottom === 0;
+    const defaultButtonAction = !buttonActionType && BUTTON_ACTION_TYPES.NEXT_PAGE;
```
Comment (GuruM) [inline src/editors/button/ButtonEditor.js]: hmm, `BUTTON_ACTION_TYPES.NEXT_PAGE` will always exist

Comment (GuruM) [reply src/editors/button/ButtonEditor.js]: what case are we really trying to address here?

Comment (helenphan0) [reply src/editors/button/ButtonEditor.js]: When people add a custom button but don't select a button action, the button does nothing. This fix will default the action to next

Comment (GuruM) [reply src/editors/button/ButtonEditor.js]: sorry i mean in this conditional in particular

**Comment by GuruM** [comment]
📌

**Pull request merged**